torch_xla/experimental/custom_kernel.py

```diff
@@ -918,9 +918,6 @@ def validate_ragged_paged_attention_inputs(
     raise ValueError(
         f"Expected {cu_q_lens.shape=} to be ({max_num_seqs + 1},)  where"
         " `max_num_seqs` is `page_indices.shape[0]`.")
-  if max_num_seqs > max_num_batched_tokens:
```

Comment:
ooc, why is it removed? They are static value and won't trigger recompilation.

Reply:
ditto